fractal-theme/assets/js/src/search-results.js

```diff
@@ -1,35 +1,43 @@
-import Fuse from 'fuse.js'
+import Fuse from "fuse.js";
```

Comment:
Did you _prettier_ all over this file? :)

Reply:
Its pretty right?

Reply:
Adding a prettier file cause I wont cause a mess :'D